docs/README.md

```diff
@@ -1,4 +1,4 @@
-# Duke
+# Omnitravel
 
 {Give product intro here}
 
```

Comment:
Very detailed diagram of how the TravelActivityList works.

Reply:
Perhaps for the second statement [command.length ==2] include an alt next to it!